AnnoDesigner/KeyRecorderWindow.xaml

```diff
@@ -0,0 +1,26 @@
+﻿<Window x:Class="AnnoDesigner.HotkeyRecordingWindow"
```

Comment:
- classname does not match filename
- File has a lot of code behind. Maybe extract to a ViewModel?

Reply:
Yeah, this was a proof of concept. I will move it to either a ViewModel or possibly its own UserControl in future.

Reply:
Ok, it was just "to note it".